bootstrap

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-BUILD_TYPE=${1:-Release}
+BUILD_TYPE="Release"
+if [ "$1" == "Release" ] || [ "$1" == "Debug" ]; then
```

Comment:
This seems weird. There are more types than just Release and Debug. Why don't we just require the parameter and always call  `BUILD_TYPE=$1; shift` If you need additional args than you have to pass in the build type as well.

Reply:
It's currently not required and 90% of the time it's probably going to be Release, so given those two things there is likely going to be a lot of mistakes if we decide to require it (e.g. it will be forgotten a lot if they do put other flags). I'm not sure the other values are worth checking for, but I can add those too.